src/components/Edit.js

```diff
@@ -33,6 +33,7 @@ https://github.com/ipfs-shipyard/peer-pad/issues/new
 // Status messages for the user to know what's going on
 const stateStatuses = {
   IDLE: 'IDLE',
+  NEEDS_SAVING: 'Needs saving...',
```

Comment:
Hm, not sure about the wording. If I open a page, do some edits and see "Needs saving...", my first instinct would be to look for a "Save" button, while "Saving" indicates that the action already started happening...

Maybe something like "About to save..." or "Saving shortly" would be better.

Reply:
The only thing about that is that it won't save if there's no pinner.

Reply:
True. Maybe we need different states here depending on if the collaboration has a pinner or not in it's membership.

Reply:
the other thing would be to add a page close event handler that warns you about unsaved changes if the save if pending or incomplete

there's kind of a matrix of states by save progress/pinner presence/changes made with values of text/status color/close handler state, should we break this out?